### PR TITLE
Async file sink

### PR DESCRIPTION
**Pull request by Zetanova**
fix of file io specs and make the file sink async or should the FileSink not be async used?

### CODE REVIEW

**Comment by Zetanova** [comment]
... was the other PR included

File: src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt [hunk 1]
```diff
@@ -59,7 +59,7 @@ namespace Akka.Cluster
     {
         public ClusterActorRefProvider(string systemName, Akka.Actor.Settings settings, Akka.Event.EventStream eventStream) { }
         protected override Akka.Actor.IActorRef CreateRemoteWatcher(Akka.Actor.Internal.ActorSystemImpl system) { }
-        public override void Init(Akka.Actor.Internal.ActorSystemImpl system) { }
```
Comment (Aaronontheweb) [inline src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt:62]: Can we move the `Akka.Cluster` changes into a separate PR?

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
Can we move the `Cluster` changes into a separate PR?

File: src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt (hunk 1 above)
Comment (Zetanova) [reply src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt:62]: yes, its in an other PR https://github.com/akkadotnet/akka.net/pull/5445, 
made a mistake with the branch, will try to resolve it

**Comment by Zetanova** [comment]
open new PR

**Pull request closed without merging**